components/form/rangeDatepicker/src/index.js

```diff
@@ -0,0 +1,128 @@
+import React, {Component, PropTypes} from 'react'
+
+import DatePicker from 'react-datepicker'
+import moment from 'moment'
+import ButtonBasic from '@schibstedspain/sui-button-basic'
+import Chevronbottom from '@schibstedspain/sui-svgiconset/lib/Chevronbottom'
+
+import 'react-datepicker/dist/react-datepicker.css'
```

Comment:
I would not include the css of the component.
We should create a scss for your component, and in it import https://github.com/Hacker0x01/react-datepicker/blob/master/src/stylesheets/datepicker.scss
IMO

Reply:
i'm agree. I can overwrite variables of the react-datepicker component